src/Sylius/Bundle/CoreBundle/Form/Type/Checkout/AddressType.php

```diff
@@ -29,32 +30,61 @@
 
 final class AddressType extends AbstractResourceType
 {
+    /** @var AddressComparatorInterface|null */
+    private $addressComparator;
+
+    public function __construct(string $dataClass, array $validationGroups = [], ?AddressComparatorInterface $addressComparator = null)
+    {
+        parent::__construct($dataClass, $validationGroups);
+
+        if (null === $addressComparator) {
+            @trigger_error(
+                sprintf(
+                    'Not passing an $addressComparator to %s constructor is deprecated since Sylius 1.7 and will be removed in Sylius 2.0.',
```

Comment:
```suggestion
                    'Not passing an $addressComparator to "%s" constructor is deprecated since Sylius 1.8 and will be impossible in Sylius 2.0.',
```